interactive-instruments/ldproxy
# Manager

Closes interactive-instruments/ldproxy-project-management#47.
Closes #264.
Closes #266.
Closes #267.
Closes #418.
Closes #419.
 

To test this PR you can run:
```
docker run --name ldproxy -d -p 7080:7080 -v ~/your_data_dir:/ldproxy/data docker.ci.interactive-instruments.de/iide/ldproxy:manager2
```
The manager will start at `http://localhost:7080/manager/`.

If you want to use a database on localhost, use the special hostname `host.docker.internal` to make it work inside the docker container (see [here](https://docs.docker.com/docker-for-mac/networking/#use-cases-and-workarounds) or [here](https://docs.docker.com/docker-for-windows/networking/#use-cases-and-workarounds)). 

Another option is to use the instance at: http://ldproxy01/manager2/manager/ (login with admin/ldproxy)

#### Work in Progress

- [x] backend sync
- [x] view codelists
- [x] view connection info
- [x] i18n + help

#### Not planned for this PR

- add|edit|delete codelists
- view|edit|delete providers
- overrides
